### ISSUE TITLE
Basic Incrementor Metrics

### ISSUE THREAD

**Issue by KoRMaK** [igorkasyanchuk/rails_performance#76]
Hello, I wanted to ask if such a feature exists and suggest it if it doesn't, a feature to capture incrementor metrics and render them to some graphs in the rails performance dashboard.

Something like what this `statsd` client offers perhaps https://github.com/reinh/statsd

**Comment by igorkasyanchuk** [comment]
how do imagine this can be upgraded?

do you want something like `RailsPerformance.metrics["some_metric"].increment` ?

you want to upgrade it from your Rails code (controllers, jobs, etc)?

**Comment by KoRMaK** [comment]
Yep, thats what I was thinking and you are correct on where the calls would be invoked. It would also be nice to view it as a time-series graph inside the rails-performance dashboard, but that may be out of scope due to complexity, so just being able to see what the current value is inside of the  dashboard would be nice.

**Comment by igorkasyanchuk** [comment]
I'll think about it
